trie/verkle_iterator.go

```diff
@@ -112,7 +112,7 @@ func (it *verkleNodeIterator) Next(descend bool) bool {
 		it.stack[len(it.stack)-1].Node = it.current
 		parent := &it.stack[len(it.stack)-2]
 		parent.Node.(*verkle.InternalNode).SetChild(parent.Index, it.current)
-		return true
```

Comment:
if a node was resolved, the function needs to be called recursively so that the iteration covers the child nodes